src/encoded/tests/test_auth0.py

```diff
@@ -1,5 +1,20 @@
 import pytest
 import requests
+from unittest import mock
+from snovault.crud_views import create_item		
```

Comment:
You're not actually using this anywhere?

Reply:
Hold over from old code. Removed.